packages/shared/src/lib/query.ts

```diff
@@ -72,6 +74,13 @@ export type AllFeedPages = SharedFeedPage | OtherFeedPage;
 
 export type MutateFunc<T> = (variables: T) => Promise<(() => void) | undefined>;
 
+export const getNextPageParam = (pageInfo: PageInfo): null | string => {
+  if (!pageInfo?.hasNextPage) {
+    return null;
+  }
+  return pageInfo?.hasNextPage && pageInfo?.endCursor;
```

Comment:
Nice wrapper 👌 Can `endCursor` be undefiend any time, maybe also return null in that case?